base_classes/NXdata.nxdl.xml

```diff
@@ -75,11 +75,12 @@
 
 		More complex cases are supported
 
-		* histogram data: ``x`` has one more value than ``data``.
-		* alternative axes: instead of a single ``x`` axis you can have several axes, one of which being the default.
-		* signals with more than one dimension: ``data`` could be 2D with axes ``x`` and ``y`` along each dimension.
-		* axes with more than one dimension: ``data`` could be 2D with axes ``x`` and ``y`` also being 2D, providing a
-		  unique ``(x, y)`` coordinate for each ``data`` point.
+		* Histogram data. For example ``x`` has one more value than ``data``.
+		* Alternative axes. For example instead of a single ``x`` axis you can have several axes, one of which being the default.
+		* Signals with more than one dimension. For example ``data`` could be 2D with axes ``x`` and ``y`` along each dimension.
+		  Equivalently 3- or higher-dimensional data is allowed.
+		* Axes with more than one dimension. For example ``data`` could be 2D with axes ``x`` and ``y`` also being 2D, providing a
+		  unique ``(x, y)`` coordinate for each ``data`` point. Equivalently 3- or higher-dimensional axes are allowed.
```

Comment:
Addresses @rayosborn comment https://github.com/nexusformat/definitions/pull/1396#pullrequestreview-2679270523

Reply:
For reference: the examples could give the impression NXdata only support 1D and 2D data.

Solve by

* Clearly prepend with "For example"
* Add "Equivalently 3- or higher-dimensional data is allowed." and "Equivalently 3- or higher-dimensional axes are allowed"